build/Makefile

```diff
@@ -127,10 +127,15 @@ install: ensure-build-image
 		agones $(mount_path)/install/helm/agones/
 
 # Build a static binary for the gameserver controller
-build-controller-binary: ensure-build-image
+build-controller-binary: ensure-build-image lint
 	docker run --rm -e "CGO_ENABLED=0" $(common_mounts) $(build_tag) go build \
 		-o $(mount_path)/cmd/controller/bin/controller -a $(go_version_flags) -installsuffix cgo $(agones_package)/cmd/controller
 
+# Lint the go source code.
+lint: ensure-build-image
+	docker run --rm $(common_mounts) -w $(mount_path) $(build_tag) bash -c \
```

Comment:
Minor nit - add $(DOCKER_RUN_ARGS)  to this line.

Reply:
Done.